content/updates/2020-07/1-rules-UI.md

```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/rules_UI.png
+pim_announcement_alt_img: "The UI to manage the rules"
+pim_announcement_audience:
+- EE
+---
+
+# A brand new UI to manage your rules
+
+In order to let everyone use the rules engine and boost your productivity, we have created a user interface to manage your rules directly in the PIM.  
+
+We know that your time is precious and that you always need to work efficiently. That's why we built the rules engine a few years ago.
```

Comment:
```suggestion
Our rules engine enables you to automate enrichment actions on specific product selections. This new screen allows you to create, update and delete rules in a few clicks, it will definitely ease and faster the rules management. 
The imports of YML files to create or update rules remains available.
```